### PR TITLE
solution

### PR DESCRIPTION
**Pull request by nrjss67**

### CODE REVIEW

**Review by vsmutok — CHANGES_REQUESTED** [review]
if you run this command a second time, it throws an error

```bash 
python create_file.py -f file.txt -d dir1 dir2
```
image

**Review by vsmutok — CHANGES_REQUESTED** [review]
if you run this command a second time, it throws an error
```bash
python create_file.py -d dir1 dir2
```
image

**Review by vsmutok — CHANGES_REQUESTED** [review]
must be an empty line if the command is executed a second time

image

image

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,41 @@
...
+import os
+import sys
+from datetime import datetime
+
+
+def file_content_printer(current_file: str) -> None:
+    with open(current_file, "a") as new_file:
+        if new_file:
+            print(file=new_file)
+        print(datetime.now().strftime("%Y-%m-%d %H:%M:%S"), file=new_file)
```
[the remark this answers is not in the document]
Comment (nrjss67) [reply app/create_file.py]: fixed.

**Pull request closed without merging**